src/server/scripts/EasternKingdoms/AlteracValley/alterac_valley.cpp

```diff
@@ -101,10 +101,10 @@ class npc_av_marshal_or_warmaster : public CreatureScript
                 events.Reset();
                 events.ScheduleEvent(EVENT_CHARGE_TARGET, urand(2 * IN_MILLISECONDS, 12 * IN_MILLISECONDS));
```

Comment:
what happens with those?
should be changed...
```events.ScheduleEvent(EVENT_CHARGE_TARGET, urand(2 * IN_MILLISECONDS, 12 * IN_MILLISECONDS));``` is the same of: ```events.ScheduleEvent(EVENT_CHARGE_TARGET, 2s, 12s);```

Reply:
Should be fixed. Thanks!